backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -724,4 +724,16 @@ public static void validateEntityNameWithColonAndSlash(String name) throws Model
           "Name can not contain ':' or '/' or '\\\\'", Code.INVALID_ARGUMENT);
     }
   }
+
+  public static ModelDBException getInvalidFieldException(IllegalArgumentException ex) {
+    if (ex.getCause() != null
+        && ex.getCause().getMessage().contains("could not resolve property: ")) {
```

Comment:
null check on ex.getCause().getMessage()

Reply:
I had added a comment on this here https://github.com/VertaAI/modeldb/pull/1524#discussion_r492474215

Reply:
Added null check https://github.com/VertaAI/modeldb/pull/1524/files#diff-678c1a74bf0f4a56c36bdfba5112c87eR730